docs/source/access-data/tutorials/initial-setup.rst

```diff
@@ -3,194 +3,189 @@ Initial Setup
+Complete the initial setup required for data access:
 
-    1. Create a new GCP project.
-    2. Create a service account and configure authentication on a local machine.
-    3. Enable the APIs on your project and install related tools locally.
+    1. :ref:`Create a new GCP project.
+       <access-data/tutorials/initial-setup:Step 1: Create a GCP project>`
+    2. :ref:`Create a service account and configure authentication on a local machine.
+       <access-data/tutorials/initial-setup:Step 2: Configure authentication>`
+    3. :ref:`Enable the APIs on your project and install related tools locally.
+       <access-data/tutorials/initial-setup:Step 3: Enable and install APIs>`
 
 Our broker lives on `Google Cloud
 Platform <https://cloud.google.com/>`__ (GCP).
 To access the data you
 need a free GCP project of your own, with a configured service account
-giving you access to Google's APIs.
+authenticating your access to Google's APIs.
 You can then connect using many different languages;
 our tutorials (except this one) demonstrate Python and command-line methods.
 
-Complete this tutorial one of two ways:
-
--   :ref:`method-a-command-line`. Install the CLI and do
-    everything from the command line.
--   :ref:`method-b-gcp-console`. Use the web Console for the GCP setup portion.
-
 This setup only needs to be done once per project/local machine.
 
-.. _method-a-command-line:
+The end of this tutorial includes instructions to
+:ref:`delete the project <delete-project>`.
 
-Method A: Command line
-----------------------
 
-Choose some parameters that will be used to create and configure your GCP project:
+Step 1: Create a GCP project
+------------------------------
 
-.. code:: bash
+Go to the Google `Cloud Resource
+Manager <https://console.cloud.google.com/cloud-resource-manager>`__ and
+login with a Google or Gmail account (go
+`here <https://accounts.google.com/signup/v2/webcreateaccount?flowName=GlifWebSignIn&flowEntry=SignUp>`__
+if you need to create one). Click "Create Project" (A). Enter a project
+name and **write down the project ID (B)**, you will need it below.
+Click "Create".
 
-    # project ID. it must be unique, so at least add a number here
-    PROJECT_ID=my-pgb-project
+.. figure:: gcp-setup.png
+   :alt: GCP setup
 
-    # service account name
-    SA_NAME=my-service-account
 
-    # local path to store your key file, ending with .json
-    KEY_PATH=/local/path/for/GCP_auth_key.json
+Step 2: Configure authentication
+---------------------------------
 
-    # nothing to choose here, just set this variable
-    SA_EMAIL="$SA_NAME@$PROJECT_ID.iam.gserviceaccount.com"
+**On GCP:**
 
+    Go to Google's "Getting started with authentication" page and complete the
+    section `Creating a service
+    account <https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account>`__
+    where you will create a service account in your project
+    and download a JSON key file for authentication.
```

Comment:
Users may wonder what minimal permission level they should assign.

Reply:
This instructions at the link tell them to create an account with the "owner" role. I updated the tutorial to confirm that is what they should do.